CHANGELOG.md

```diff
@@ -1,3 +1,65 @@
+## 1.6.0 / 2017-04-14
+
+* [CHANGE] Replaced the **experimental** remote write implementations for
+  various backends by a generic write interface with example bridge
+  implementation for various backends.
+* [FEATURE] New flag `-storage.local.target-heap-size` to tell Prometheus about
+  the desired heap size. This deprecates the flags
+  `-storage.local.memory-chunks` and `-storage.local.max-chunks-to-persist`,
+  which are kept for backward compatibility.
+* [FEATURE] Add `check-metrics` to `promtool` to lint metric names.
+* [FEATURE] Auto-tweak times between checkpoints to limit time spent in
```

Comment:
Isn't that more of an ENHANCEMENT if it's just something that now transparently works better in the background (haven't looked at it closely)?